### PR TITLE
[17.09] Correct base route for workflows, allowing proxy-prefix to work.

### PR DESCRIPTION
**Pull request by dannon**
WIP while I fix a few more of this same class of error.

This fixes the bug discussed in https://biostar.usegalaxy.org/p/24988/

### CODE REVIEW

**Comment by jmchilton** [comment]
... ... ... we should probably just run our Selenium test suite with a proxy-prefix on Jenkins when we run it huh?

**Comment by martenson** [comment]
@jmchilton hell yeah

**Comment by dannon** [comment]
I was working on refactoring a bit to have a single utils method for prepending the base URL (and ensuring Galaxy.root exists), but I'm going to pop that change over to dev instead.

**Comment by jmchilton** [comment]
@dannon Awesome - thanks for the fixes!

**Comment by mandorodriguez** [comment]
Ok did a pull and the page with workflows works again. Thanks!

**Comment by dannon** [comment]
@mandorodriguez Great!  Happy to help.

File: client/galaxy/scripts/onload.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ $(document).ready( function() {
     }
 
     // Load all webhooks with the type 'onload'
-    $.getJSON('/api/webhooks/onload/all', function(webhooks) {
+    $.getJSON( Galaxy.root + '/api/webhooks/onload/all', function(webhooks) {
```
Comment (guerler) [inline client/galaxy/scripts/onload.js:188]: Should be `api...` instead of `/api...` right?

Comment (dannon) [reply client/galaxy/scripts/onload.js:188]: Both actually work, but I'd rolled that fix into my util function refactoring of this.  If you want, I'll PR that change here for consistency.

Comment (dannon) [reply client/galaxy/scripts/onload.js:188]: (PR'd at #4715; might as well be consistent even with it being rewritten in -dev I guess)

**Comment by mandorodriguez** [comment]
Posted on biostars but guess I'll ask here too. My users existing workflows showed up after this but when she opens the workflow the sentence that is under the headings that explains what it is gone. Also names of the headings just say input where as before she had it say what it was (e.x reference genome). Is that text there and just not displaying?

**Pull request merged**